lib/auth.ts

```diff
@@ -0,0 +1,29 @@
+import dbConnect from "@/utils/mongodb";
+import User from "@/models/User";
+import type { NextAuthOptions } from "next-auth";
+import credentials from "next-auth/providers/credentials";
+import bcrypt from "bcryptjs";
+
+export const authOptions: NextAuthOptions = {
+  providers: [
+    credentials({
+      name: "Credentials",
+      id: "credentials",
+      credentials: {
+        email: { label: "Email", type: "text" },
+        password: { label: "Password", type: "password" },
+      },
+      async authorize(credentials) {
+        await dbConnect();
+        const user = await User.findOne({ email: credentials?.email }).select("+password");
+        if (!user) throw new Error("Wrong Email");
```

Comment:
from security point of view it is better to show error `"Wrong email and/or password"`